src/Symfony/Component/Console/Helper/TableStyle.php

```diff
@@ -108,18 +117,71 @@ public function getVerticalBorderChar()
         return $this->verticalBorderChar;
     }
 
+    /**
+     * Sets crossing characters.
+     *
+     * Example:
+     * <code>
```

Comment:
We are not using HTML in desc, so this should be removed.

Reply:
`<code>` tag is sometimes used yes (on master, 17 occurrences). This code-description can be useful IMO.